claudiu-website-main-page.css

```diff
@@ -0,0 +1,140 @@
+* {
+    margin: 0;
+    padding: 0;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+/* 
+body {
+    display: flex;
+    flex-direction: column;
+
+} */
+
+.intro {
+    background-image: linear-gradient(rgba(0,0,0,0.1),rgba(0,0,0,0.1)), url(https://scontent-lcy1-1.xx.fbcdn.net/v/t31.18172-8/17158872_10155091890419629_4504477491907991475_o.jpg?_nc_cat=102&ccb=1-3&_nc_sid=09cbfe&_nc_ohc=WXsr67K4ES8AX_uZQBr&_nc_ht=scontent-lcy1-1.xx&oh=ebbef15e582660ca564808e30883861c&oe=608BD7CB);
+    min-height: 100vh;
+    background-size: cover;
+    background-position: center;
+}
+
+ul {
+    float: right;
+    list-style-type: none;
+    margin-top: 25px;
+    margin-left: -60px;
+    overflow: hidden;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    padding: 14px 16px;
+}
+
+ul li{
+    display: inline-block;
+    float: right;
+}
+
+ul li a{
+    text-decoration: none;
+    color: #fff;
+    padding: 14px 20px;
+    border: 1px solid transparent;
+    transition: 0.6s ease;
+    
+}
+
+ul li a:hover{
+    background-color: #fff;
+    color: #000;
+}
+
+ul li.active a {
+    background-color: #fff;
+    color: #000;
+}
+
+.logo img {
+    float: left;
+    width: 70px;
+    height: auto;
+}
+
+
+
+.title{
+    position: absolute;
+    top: 50%;
+    left:50%;
+    transform: translate(-50%, -50%);
+}
+
+.title p{
+    color: #fff;
+    font-size: 30px;
+    text-align: center;
+    margin-top: 20%;
+    visibility: hidden;
+}
+
+.title h1{
+    color: #fff;
+    font-size: 35px;
+    margin-bottom: -300px;
+    font-weight: 700;
+    margin-top: -5%;
+    border: 5px solid #fff;
+    border-width: 5px;
+    padding: 10px 30px;
+    width: -10px;
+    font-style: italic;
+    text-align: center;
+    text-shadow: 2px 2px 4px #000000;
+    
+}
+
+.button {
+    position: absolute;
+    top: 91.5%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.btn {
+    border: 1px solid #fff;
+    padding: 10px 30px;
+    color: #fff;
+    text-decoration: none;
+    transition: 0.6s ease;
+
+}
+
+.btn:hover {
+    background-color: #fff;
+    color: #000;
+}
+
+
+footer {
+    text-align: center;
+    font-size: 80%;
+    clear: both;
+    position: relative;
+    background: #FFFFFD;
+    margin-top: -15px;
+}
+
+.main {
+    max-width: 1200px;
```

Comment:
Rather than using an absolute width in px here, what other method could you have used so that the page displays correctly on different sized display screens?